### PR TITLE
SSH compression

### PR DESCRIPTION
**Pull request by pivnicek**
Add ssh compression.

Waiting on https://github.com/keboola/db-extractor-common/pull/106

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/PgSQL.php [hunk 1]
```diff
@@ -214,12 +220,14 @@ protected function executeCopyQuery(string $query, CsvFile $csvFile, string $tab
...
             )
         );
+        echo "\mCOMMAND: $command\m";
 
         $process = new Process($command);
         // allow it to run for as long as it needs
         $process->setTimeout(null);
         $process->run();
         if (!$process->isSuccessful()) {
+            var_export($process->getErrorOutput());
```
Comment (MiroCillik) [inline src/Keboola/DbExtractor/Extractor/PgSQL.php]: Is this `var_export` necessary? The error output is logged one line below.

File: src/Keboola/DbExtractor/Extractor/PgSQL.php [hunk 2]
```diff
@@ -214,12 +220,14 @@ protected function executeCopyQuery(string $query, CsvFile $csvFile, string $tab
                 )
             )
         );
+        echo "\mCOMMAND: $command\m";
```
Comment (MiroCillik) [inline src/Keboola/DbExtractor/Extractor/PgSQL.php]: Why not use logger for this?

File: src/Keboola/DbExtractor/Extractor/PgSQL.php (hunk 1 above)
Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/PgSQL.php]: sorry, debug, removing.  thanks!

File: src/Keboola/DbExtractor/Extractor/PgSQL.php (hunk 2 above)
Comment (pivnicek) [reply src/Keboola/DbExtractor/Extractor/PgSQL.php]: likewise, thanks man

**Comment by pivnicek** [comment]
tak uz @MiroCillik co ty na to?

**Comment by pivnicek** [comment]
The decimal values are different now because of the phpunit upgrade.  The old values worked out as equal in the old assertion but in the new phpunit the assertion is more strict.

**Pull request merged**